plugins/ssh-plugin/package.json

```diff
@@ -0,0 +1,43 @@
+{
+      "name": "@eclipse-che/theia-ssh-plugin",
+      "keywords": [
+        "theia-plugin"
+      ],
+      "version": "0.0.1",
+      "license": "EPL-2.0",
+      "contributors": [
+        {
+          "name": "Artem Zatsarynnyi",
+          "email": "azatsary@redhat.com"
+        },
+        {
+          "name": "Igor Vinokur",
+          "email": "ivinokur@redhat.com"
+        }
+      ],
+      "files": [
+        "src"
+      ],
+      "devDependencies": {
+        "@theia/plugin": "0.3.16",
```

Comment:
could you please set the latest (which is 0.3.18)

Reply:
ok